apps/common-app/src/examples/RuntimeTests/tests/animations/withSequence/colors.test.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect } from 'react';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  withSequence,
+  withDelay,
+} from 'react-native-reanimated';
+import {
+  describe,
+  test,
+  render,
+  wait,
+  useTestRef,
+  getTestComponent,
+  expect,
+} from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+import { View, StyleSheet } from 'react-native';
+import { ComparisonMode } from '../../../ReanimatedRuntimeTestsRunner/types';
+
+type TestCase = {
+  startColor: string;
+  middleColor: string;
+  finalColor: string;
+};
+
+describe('withSequence animation of number', () => {
+  enum Component {
+    ACTIVE = 'ONE',
+    PASSIVE = 'TWO',
+  }
+  const DELAY = 75;
+  const WidthComponent = ({ startColor, middleColor, finalColor }: TestCase) => {
+    const colorActiveSV = useSharedValue(startColor);
+    const colorPassiveSV = useSharedValue(startColor);
+
+    const refOne = useTestRef(Component.ACTIVE);
+    const refTwo = useTestRef(Component.PASSIVE);
```

Comment:
Wouldn't it be better to name those refs according to what they're holding?
e.g `refActive`, `refPassive`